crates/noirc_evaluator/src/ssa/acir_gen/operations/intrinsics.rs

```diff
@@ -235,6 +235,25 @@ fn prepare_outputs(
     outputs
 }
 
+fn prepare_outputs_no_count(
```

Comment:
This new method `prepare_outputs_no_count` was created with the addition of variable output sizes. It feels a bit hacky, but all integration tests do pass. Do we think this is fine for now or should I add logic to expose backend specific information on variable output sizes of blackbox funcs?